### DISCUSSION TITLE
In Dock staging Not able to create Observation with rubrics Question set.

### DISCUSSION THREAD

**Discussion by ramkumar-ns-dev** [Sunbird-coKreat/Community#91] [General Discussions]
### **Prerequisites**
I have searched for existing issues that already report this problem, without success.

### **Current Behaviour**
I'm trying to create complete flow for Observation with rubrics. I can able to create Project with TPD framework and Observation with rubrics primary category. And Contributor also able to nominate to the project. But when contributor try to create/upload question set  using **create/upload content** flow, data is not populating on branching logic and it's breaking the application.

### **Expected Behaviour**
While the Contributor create/upload the question set, framework data will be populated and contributor able to add question in the Question set structure, then submit to review to reviewer.

### **Steps to Reproduce**
	1. As a creator a publish new project with TPD framework and category of Observation with rubrics
	2. Login to contributor flow and Nominate the published project.
	3. In Creator page approve the Nomination.
	4. Again in Contributor page, Please click on **create/upload content** for the particular project. Editor page with open with broken UI (In console Errors will shown).
	
### **API Endpoints used in create/upload flow**
        1. https://dock.sunbirded.org/content/questionset/v1/read/do_113850820441710592112?mode=edit
        2. https://dock.sunbirded.org/action/questionset/v1/hierarchy/do_113850820441710592112?mode=edit
        3. https://dock.sunbirded.org/action/questionset/v1/read/do_113850820441710592112?mode=edit&fields=instructions
        
### Additional Information
Tested on Google Chrome: Version 114.0.5735.198 (Official Build) (arm64)

can anybody help on this.
cc: @kiranharidas187

**Comment by amitpriyadarshi** [comment]
@ramkumar-pacewisdom as per discussion issue was with respect to api-key. Please cross check the questionset read hierarchy api with corrected api-key.

cc: @Krishnaj20

**Reply by ramkumar-ns-dev** [reply]
sure, will check with the api-key
